modular_zubbers/code/datums/quirks/positive_quirks/venemous_bite/venomous_bite_quirk.dm

```diff
@@ -0,0 +1,86 @@
+/// Format: (reagent typepath -> list(amount to inject per bite, cooldown))
+GLOBAL_LIST_INIT(venomous_bite_choice_specs, list(
+	/datum/reagent/toxin/player_venom = list(5, 80 SECONDS),
+	/datum/reagent/toxin/venom = list(5, 180 SECONDS),
+	/datum/reagent/toxin/staminatoxin = list(5, 120 SECONDS),
+	/datum/reagent/toxin/carpotoxin = list(5, 60 SECONDS), // less powerful than the zootoxin
+	// medicine
+	/datum/reagent/medicine/epinephrine = list(3, 20 SECONDS),
+	/datum/reagent/medicine/omnizine = list(2, 30 SECONDS), // barely worth it due to the damage biting does
+	// drugs
+	/datum/reagent/drug/space_drugs = list(5, 60 SECONDS),
```

Comment:
I forgot to add space drugs and mindbreaker - these are hardly harmful, so balance shouldnt be a concern